### PR TITLE
AB#14162 - Windows build

### PR DESCRIPTION
**Pull request by Krizai**

### CODE REVIEW

**Review by kgominyuk — APPROVED** [review]

File: build-scripts/Windows/Build-Zip.ps1 [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+$ZlibDependencySourceUrl = "https://spark-prebuilt-binaries.s3.amazonaws.com/zlib.zip"
+$ZlibDependencyDir = "zlib"
+$ZlibDependencyPath = "$DependenciesPath\$ZlibDependencyDir\zlib-win32-1"
+$MiniZipDependencyPath = "$DependenciesPath\minizip"
+
+$S3Key = $env:SPARK_PREBUILT_KEY
+if (!$S3Key) {
+    Write-Host "Sprak prebuilt storage key(SPARK_PREBUILT_KEY) is required"
+    Exit
```
Comment (lxbndr) [inline build-scripts/Windows/Build-Zip.ps1]: We are exiting with success here. Probably we should indicate an error.
> To declare a terminating error, use the Throw keyword. For more information, see [about_Throw](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_throw?view=powershell-7.1).

Also, typo in "Sprak"

File: build-scripts/Windows/Build-Zip.ps1 [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+$HeaderSearchPaths = "$InstallPath\include" 
+$LibrarySearchPaths = "$InstallPath\lib"
+
+$ZlibDependencySourceUrl = "https://spark-prebuilt-binaries.s3.amazonaws.com/zlib.zip"
+$ZlibDependencyDir = "zlib"
+$ZlibDependencyPath = "$DependenciesPath\$ZlibDependencyDir\zlib-win32-1"
+$MiniZipDependencyPath = "$DependenciesPath\minizip"
+
+$S3Key = $env:SPARK_PREBUILT_KEY
+if (!$S3Key) {
```
Comment (lxbndr) [inline build-scripts/Windows/Build-Zip.ps1]: I know that '!' is a shortcut for '-not', but would be good to use consistent style across our build scripts

File: build-scripts/Windows/Build-Zip.ps1 [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    
+    ModuleName = $ModuleName
+
+    WorkPath = $ProjectRoot
+
+    IntermediatesPath = $IntermediatesPath
+    ProductsPath = $ProductsPath
+
+    SwiftVersion = 5
+    EnableTesting = $true
```
Comment (lxbndr) [inline build-scripts/Windows/Build-Zip.ps1]: Testing is not required, I guess. Should be `$false`.

**Review by lxbndr — CHANGES_REQUESTED** [review]

File: build-scripts/Windows/Build-Zip.ps1 (hunk 3 above)
Comment (Krizai) [reply build-scripts/Windows/Build-Zip.ps1]: Agree

File: build-scripts/Windows/Build-Zip.ps1 (hunk 2 above)
Comment (Krizai) [reply build-scripts/Windows/Build-Zip.ps1]: Didn't notice this, thank you

File: build-scripts/Windows/Build-Zip.ps1 (hunk 1 above)
Comment (Krizai) [reply build-scripts/Windows/Build-Zip.ps1]: Thank you. Changed

**Review by lxbndr — APPROVED** [review]

**Pull request merged**